lib/services/lwm2mHandlers/updateRegistration.js

```diff
@@ -19,12 +19,15 @@
  *
```

Comment:
CNR entry missing

Reply:
I always forget this... sorry!!

Reply:
Fixed in b5f4e48a88f7551ff47ca0257b3d620345d64130